doc/contribute/devtheory.rst

```diff
@@ -202,3 +202,112 @@ the known current volume, the next volume is determined by looking at the other
 volume tagged on that surface (as described in the Sense Tags section above).
 This assumes that a valid surface and volume are provided. If no next volume
 exists, then the call will return 0 for the next volume.
+
+DAGMC Metadata
+~~~~~~~~~~~~~~
+
+DAGMC's Metadata Interface
```

Comment:
I feel as though this section may be better coming after the next section.  That is, first describe the data in file and then describe the first s/w interface to access this data.

Reply:
With that change, you can also include a phrase/sentence in this section about the fact that this data is unpacked from the MOAB entity sets described in the other section.

Reply:
I like it. I'll make that change shortly.